browser/socket.js

```diff
@@ -56,24 +55,35 @@ socket.on('usersUpdated', users => {
     // These values are passed up as "scene" in the user tick and correspond to the names of react components and a-scenes
     const currentScene = window.location.pathname.replace(/\//g, '') || 'root';
     // If the user is on the current scene, add or update the user's head and body
-    const avatarHead = document.getElementById(user.get('id'));
-    const avatarBody = document.getElementById(`${user.get('id')}-body`);
     if (user.get('scene') === currentScene) {
-      // console.log(`Attempting to update ${user.get('id')}-body`);
+      const avatarHead = document.getElementById(user.get('id'));
+      const avatarBody = document.getElementById(`${user.get('id')}-body`);
```

Comment:
which it looks like I might have already done. Going to mess around a bit with this.